app/controllers/admin/publishers_controller.rb

```diff
@@ -92,4 +94,9 @@ def update_params
   def sortable_columns
     [:last_sign_in_at, :created_at, Publisher::VERIFIED_CHANNEL_COUNT]
   end
+
+  def is_a_uuid?(uuid)
+    uuid_regex = /^[0-9a-f]{8}-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{12}$/
```

Comment:
if you didn't write this, we should probably credit where it came from in a comment

Reply:
Ah good point - added the reference to the original location
https://github.com/brave-intl/publishers/pull/1538/commits/7deafa87df6a7870871f1f2ec588999d5c85e6b9